src/services/ApiService.ts

```diff
@@ -14,7 +14,7 @@ axiosApiInstance.interceptors.response.use(
   (res: AxiosResponse) => res,
   async (error: unknown) => {
     if (error instanceof AxiosError && error.response?.status === HttpStatusCode.Unauthorized) {
-      return `/api/frontend/login?redirectUrl=${location.href}`
+      return `/api/auth/login?redirectUrl=${location.href}`
```

Comment:
Wie erfährt das Frontend, dass es bei /auth an den KC gehen soll? Falls das über das Backend geht: In den API Docs von localhost:9090 gibt es keinen /auth Endpoint

Reply:
Der Endpunkt unter /api/auth/login schickt einen redirect. Ist genau wie vorher, nur umbenannt (Die Änderungen sind noch nicht auf Main, da ich beide zusammen mergen wollte https://github.com/dBildungsplattform/dbildungs-iam-server/pull/224)

Reply:
Das war mein Fehler. Wenn man das Backend lokal mit docker compose startet, wird nicht der lokale Stand gestartet, sondern main.